src/randoop/JunitFileWriter.java

```diff
@@ -36,10 +35,12 @@
 
   private Map<String, List<List<ExecutableSequence>>> createdSequencesAndClasses = new LinkedHashMap<String, List<List<ExecutableSequence>>>();
 
+  private Map<String, Integer> methodCountMap = new LinkedHashMap<String,Integer>();
+
   public JunitFileWriter(String junitDirName, String packageName, String junitDriverClassName, int testsPerFile) {
     this.dirName = junitDirName;
-    this.packageName = packageName;
-    this.junitDriverClassName = junitDriverClassName;
+    this.packageName = packageName.trim();
```

Comment:
I think we should be able to assume that by the time the package name gets here that it is clean. But probably should ensure that is true wherever the name comes from.